packages/syntax-core/src/TextField/TextField.tsx

```diff
@@ -80,6 +81,10 @@ export default function TextField({
    * Value of the TextField
    */
   value: string;
+  /**
+   * Specified legal number intervals for an input field. Specifically for time or number. If for time specify for milliseconds. Must be a positive value.
```

Comment:
nit: a bit hard to read
```suggestion
   * Specified legal number intervals for an input field. Specifically for time or number. If for time, specify in milliseconds. Must be a positive value.
```